### DISCUSSION TITLE
inconsistent spacing between images

### DISCUSSION THREAD

**Discussion by KarlMaoChina** [TorchIO-project/torchio#973] [Q&A] [answered]
hi , I am new to Torchio, I am trying to process a CT dataset with a varying spacing between images (of course the spacing between image and label are the same). I get a error message when trying to load the image saying the check failed as 

RuntimeError: More than one value for "spacing" found in subject images:
{'label': (1.0, 1.0, 1.0), 't1': (4.609375, 4.609375, 5.762500286102295)}
(the second one are the correct one as most of the CT data are resized to a same size 64*64*64 without keeping the spacing same between CT)

is that a expected behavior or what have I modify to make sure the dataloader work? I think I need to do something to make sure the 'label' get the correct spacing.
thank you 

update: sorry I see the error here. I accidently removed the spacing information in the label file

**Comment by fepegar** [comment, marked as answer]
I see you've found the problem. Locking this discussion.
